packages/augur-ui/src/modules/alerts/selectors/alerts.ts

```diff
@@ -24,7 +24,7 @@ export const selectInfoAlertsAndSeenCount = createSelector(
         .filter(
           alert =>
             (alert.networkId === networkId.toString() &&
-              alert.universe === universe.id) ||
+              alert.universe === universe.id && (alert.status === "Failure" || alert.status === "Success")) ||
```

Comment:
need constants for Failure and Success

Reply:
was causing a crash when i used those but /constants has some similar ones